code/game/objects/items/devices/mat_synth.dm

```diff
@@ -33,6 +33,29 @@
 	var/list/cant_scan = list()
 	var/matter = 0
 
+/obj/item/device/material_synth/preloaded
+	matter = MAX_MATSYNTH_MATTER
+
+/obj/item/device/material_synth/preloaded/admin
+	materials_scanned = list("metal" = /obj/item/stack/sheet/metal,
+							 "glass" = /obj/item/stack/sheet/glass/glass,
+							 "reinforced glass" = /obj/item/stack/sheet/glass/rglass,
+							 "plasteel" = /obj/item/stack/sheet/plasteel,
+							 "plasma glass" = /obj/item/stack/sheet/glass/plasmaglass,
+							 "reinforced plasma glass" = /obj/item/stack/sheet/glass/plasmarglass,
+							 "silver" = /obj/item/stack/sheet/mineral/silver,
+							 "gold" = /obj/item/stack/sheet/mineral/gold,
+							 "diamond" = /obj/item/stack/sheet/mineral/diamond,
+							 "plasma" = /obj/item/stack/sheet/mineral/plasma,
+							 "uranium" = /obj/item/stack/sheet/mineral/uranium,
+							 "plastic" = /obj/item/stack/sheet/mineral/plastic,
+							 "mythril" = /obj/item/stack/sheet/mineral/mythril,
+							 "bananium" = /obj/item/stack/sheet/mineral/clown,
+							 "phazon" = /obj/item/stack/sheet/mineral/phazon,
+							 "telecrystal" = /obj/item/stack/sheet/mineral/telecrystal,
+							 "sandstone" = /obj/item/stack/sheet/mineral/sandstone,
+							 "bricks" = /obj/item/stack/sheet/mineral/brick)
+
```

Comment:
`matter = INFINITY`
This would help with testing.

Reply:
Matter isn't even used at all on the admin ones, so this is pointless
Also there's no max matter var

Reply:
I had meant to post matter = infinity. It doesn't use matter? You learn something new.